CHANGELOG.md

```diff
@@ -1,3 +1,80 @@
+## v0.27.0 (2024-04-19)
+
+>  **Note:**
+
+>  - Filter v1 protocol and its REST-API access have been deprecated.
+>  - A new field of the `WakuMetadataRequest` protobuf for shards was introduced. The old shards field (2) will be deprecated in 2 releases time
+> - CLI flags `--requestRateLimit` and `requestRatePeriod` have been added for rate limiting configuration. 
```

Comment:
```suggestion
>  - CLI flags `--requestRateLimit` and `--requestRatePeriod` have been added for rate limiting configuration. Period is measured in seconds. Limits are measured per protocol per period of time. Over limit will result in TOO_MANY_REQUEST (429) response.
```

Reply:
Ooh amazing! Thanks for the improvement! 😍